unison-cli/src/Unison/CommandLine/InputPatterns.hs

```diff
@@ -1086,9 +1086,9 @@ push =
                 "publishes the current namespace into the remote namespace `remote`"
               ),
               ( "`push`",
-                "publishes the current namespace"
-                  <> "into the remote namespace configured in `.unisonConfig`"
-                  <> "with the key `GitUrl.ns` where `ns` is the current namespace"
+                "looks up the remote namespace configured in `.unisonConfig`"
+                  <> "at the key `RemoteMappings.<namespace>` where `<namespace>` is the current namespace,"
+                  <> "then publishes the current namespace to that location."
```

Comment:
I think we want to keep the help text for `push` to have its emphasis on "pushing"/"publishing" (like before) vs emphasizing the remote namespace lookup. — Same for `pull`, above.

Reply:
"publishes the current namespace into its associated remote namespace in .unisonConfig"

or
"publishes the current namespace into the remote namespace configured in `.unisonConfig` at the key `RemoteMappings.<namespace>` where `<namespace>` is the current namespace.

We should probably add a separate help page and/or help topic for `remote-mappings`, but that can be a separate ticket.